### PR TITLE
[ SERVICE-224 ] Move Storage Layer in S&R from the Provider to the Manager

### PR DESCRIPTION
**Pull request by MichaelMCoates**

### CODE REVIEW

**Comment by MichaelMCoates** [comment]
Good catch, updated.

File: src/demo/LayoutsUI.ts [hunk 1]
```diff
@@ -19,7 +20,8 @@ window.forgetMe = forgetMe;
 export async function setLayout() {
     const name = (document.getElementById('layoutName') as HTMLTextAreaElement).value;
     const layoutSelect = document.getElementById('layoutSelect') as HTMLSelectElement;
-    const layout = await Layouts.saveCurrentLayout(name);
+    const layout = await Layouts.generateLayout();
+    layout.name = name;
```
Comment (pjbroadbent) [inline src/demo/LayoutsUI.ts]: Should we remove `name` from the `Layout` object? We no longer use it anywhere in the service, and clients will likely want to manage how they identify layouts in their own way.

Comment (MichaelMCoates) [reply src/demo/LayoutsUI.ts]: Yep, let's do it.

**Comment by MichaelMCoates** [comment]
Removed `name` from the Layout interface. Moved to a Workspace interface

**Comment by tomrobertsOF** [comment]
My review still stands. Happy to merge it.

**Pull request merged**